library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -38,22 +39,30 @@
 
     private final ContentResolver resolver;
     private final DownloadDeleter downloadDeleter;
+    private final Uri batchContentUri;
+    private final Uri allDownloadsContentUri;
 
-    public BatchRepository(ContentResolver resolver, DownloadDeleter downloadDeleter) {
+    public static BatchRepository newInstance(ContentResolver resolver, DownloadDeleter downloadDeleter) {
```

Comment:
we aren't using this pattern in this project. Granted I can see how here it stops the duplication of using the constants in multiple places. So i'm happy either way